vagga.yaml

```diff
@@ -2,25 +2,83 @@ containers:
   pip-freeze:
     setup:
       - !Alpine v3.4
+      - !BuildDeps
+        - libpq
       - !Py3Install
         - pip
         - 'Django >=1.9,<1.10'
         - 'gunicorn >=19.5,<19.6'
         - 'whitenoise >=3.2,<3.3'
         - 'django-environ >=0.4,<0.5'
+        - 'psycopg2 >=2.6,<2.7'
       - !Sh pip freeze > requirements.txt
   django:
     setup:
       - !Alpine v3.4
       - !PipConfig
         dependencies: true
+      - !Install
+        - libpq
       - !Py3Requirements requirements.txt
+    environ:
+          DATABASE_URL: postgresql://vagga:vagga@127.0.0.1:5433/mainDB
+  postgres:
+    setup:
+    - !Ubuntu xenial
+    - !EnsureDir /data
+    - !Sh |
+        addgroup --system --gid 200 postgres
+        adduser --uid 200 --system --home /data --no-create-home \
+            --shell /bin/bash --group --gecos "PostgreSQL administrator" \
+            postgres
+    - !Install [postgresql-9.5]
+    environ:
+      PGDATA: /data
+      PG_PORT: 5433
+      PG_DB: mainDB
+      PG_USER: vagga
+      PG_PASSWORD: vagga
+      PG_BIN: /usr/lib/postgresql/9.5/bin
+    volumes:
+      /data: !Persistent
+        name: postgres
+        owner-uid: 200
+        owner-gid: 200
+        init-command: _pg-init
+      /run: !Tmpfs
+        subdirs:
+          postgresql: { mode: 0o777 }
 commands:
-  run: !Command
-    description: Start server
-    container: django
-    run: python3 manage.py runserver
+  _pg-init: !Command
+    description: Init postgres database
+    container: postgres
+    user-id: 200
+    group-id: 200
+    run: |
+      set -ex
+      ls -la /data
+      $PG_BIN/pg_ctl initdb
+      $PG_BIN/pg_ctl -w -o '-F --port=$PG_PORT -k /tmp' start
```

Comment:
I've got fully clear project. But after add and reboot - new post still there, it's ok?